pkgs/development/python-modules/flet-runtime/default.nix

```diff
@@ -10,14 +10,20 @@
 
 buildPythonPackage rec {
   pname = "flet-runtime";
-  version = "0.18.0";
+  inherit (flet-client-flutter) version src;
   pyproject = true;
 
-  src = fetchPypi {
-    pname = "flet_runtime";
-    inherit version;
-    hash = "sha256-VfPTfCJXpRZsKM4ToFyl7zxbk58HT6eOYthfzAM4f88=";
-  };
+  sourceRoot = "${src.name}/sdk/python/packages/flet-runtime";
+
+  postPatch = ''
+    cat <<EOF > src/flet_runtime/_setup_runtime.py
+    import os
+    if 'FLET_VIEW_PATH' not in os.environ:
+      os.environ["FLET_VIEW_PATH"] = "${flet-client-flutter}/bin"
+    EOF
+
+    echo -e "import flet_runtime._setup_runtime\n$(cat src/flet_runtime/__init__.py)" > src/flet_runtime/__init__.py
+  '';
```

Comment:
I think an actual patch might make this easier to read, i.e. using `patches`.

Reply:
I did an auxiliar file + `substitute`

No inline code, but I had to make sure that the import is the first line